modules/navier_stokes/src/fvbcs/INSFVWallFunctionBC.C

```diff
@@ -48,11 +44,12 @@ INSFVWallFunctionBC::computeStrongResidual()
   // Get the velocity vector
   const FaceInfo & fi = *_face_info;
   const Elem & elem = fi.elem();
-  ADRealVectorValue velocity(_u_var->getElemValue(&elem));
-  if (_v_var)
-    velocity(1) = _v_var->getElemValue(&elem);
-  if (_w_var)
-    velocity(2) = _w_var->getElemValue(&elem);
+  Moose::ElemArg elem_arg{&elem, false};
```

Comment:
you should retrieve the skewness argument from the variable?

Reply:
I guess we dont have the variables anymore. You d have to query the variable from intialSetup and keep the skewness argument in a class member

Reply:
I think @grmnptr expressed interest in having this be controlled more on an operator level as opposed to a global level, but I could be mis-remembering

Reply:
I ll wait on Peter but essentially I m at an approval point here

Reply:
Yes I'll do these once I get to the point where unify limiting/interpolation. I think it would be best to do it on a per-kernel basis.